extraTs/index.ts

```diff
@@ -0,0 +1,43 @@
+import {displayPyramid} from './patterns/./diamond';
```

Comment:
when i am doing this according to your suggestion it will start show an error.

Reply:
it will give you an error because you are importing displayPyramid as default but for this you have to export as default as well. So check that

- Please read about es6 import export properly and diff b/w default import/export and normal import/export.